app/CreateDeal.tsx

```diff
@@ -7,19 +7,22 @@ import { PageHeader } from "./ui/PageHeader";
 import { FormLabel } from "./ui/FormLabel";
 import { FormGroup } from "./ui/FormGroup";
 import { FormControlString } from "./ui/FormControlString";
-import { Form, FormCheck, InputGroup } from "react-bootstrap";
+import { InputGroup } from "react-bootstrap";
 import { FormControlNumber } from "./ui/FormControlNumber";
+import { FormCheckRadio } from "./ui/FormCheckRadio";
 
 export function CreateDeal(props: {
   onCreateDealClick: () => void;
   onCancelClick: () => void;
 }): ReactElement {
+  const units = ["kg", "L", "unit"];
```

Comment:
constants that actually never change should live outside the component, otherwise they'll be regenerated with every render. Also since we're working in typescript, we can have an enum for these values instead of having them be free strings

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/30/commits/c435a672ec4154dddd28edd6d79d2117417ff582